lib/molecules/admin-bar/example.html

```diff
@@ -0,0 +1,26 @@
+<div class="{{class}}" ln-m-admin-bar ln-load-from-api="false"
+  ln-site-name="{{lnSiteName}}" ln-user-name="{{lnUserName}}" 
+  ln-dashboard-url="{{lnDashboardUrl}}" ln-logout-url="{{lnLogoutUrl}}" 
+  ln-edit-page-url="{{lnEditPageUrl}}" ln-current-post-id="{{lnCurrentPostId}}" 
+  ln-new-item-types="lnNewItemTypes">
+</div>
+
+<!--
+OR... IF YOU WANT TO LOAD CONFIGURATION FROM WP-API, JUST USE IT LIKE THIS:
+
+<div class="{{class}}" ln-m-admin-bar ln-load-from-api="true" ln-current-post-id="{{lnCurrentPostId}}"></div>
+
+<script>
+  angular
+    .module('app', [
+      'lnPatterns'
+    ])
+    .run(appRun);
+
+  appRun.$inject = ['lnMAdminBarService'];
+
+  function appRun(lnMAdminBarService) {
```

Comment:
Yes, it's done in the home template of the update-patternlab branch, but we can move it to the header you created. https://github.com/moxie-lean/ng-web-app/blob/update-patternlab/app/custom/templates/home/template.html